.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
```

Comment:
>  (Please, remove the unnecessary notes, including this one.)

I've seen projects using HTML comments to give instructions. This way we wouldn't have to ask them to remove anything.

For example:

```

## Steps to reproduce

<!-- A description of how to trigger this bug. -->
```

Reply:
Nice idea, done! Thanks :)